xep-0424.xml

```diff
@@ -167,7 +170,7 @@
     <forwarded xmlns='urn:xmpp:forward:0'>
       <delay xmlns='urn:xmpp:delay' stamp='2019-09-20T23:08:25Z'/>
       <message type="groupchat" from="romeo@montague.example" to="lord@capulet.example" id="wrong-recipient-1">
-        <retracted stamp='2019-09-20T23:09:32Z' xmlns='urn:xmpp:message-retract:0'>
+        <retracted stamp='2019-09-20T23:09:32Z' xmlns='urn:xmpp:message-retract:1'>
           <origin-id xmlns='urn:xmpp:sid:0' id='origin-id-1'/>
```

Comment:
This should probably be changed to using a stanza-id instead of origin-id in cases where the MUC assigned stanza-id was used.

Reply:
The id should probably just be an attribute on `<retracted/>` called `id` and then depending on context it easier refers to the stanza-id or the message id.

Reply:
> The id should probably just be an attribute on <retracted/> called id and then depending on context it easier refers to the stanza-id or the message id.

Done.